cmd/cstor-pool-mgmt/controller/replica-controller/handler.go

```diff
@@ -160,13 +166,6 @@ func (c *CStorVolumeReplicaController) cVRAddEventHandler(cVR *apis.CStorVolumeR
 	} else {
 		common.SyncResources.Mux.Unlock()
 	}
-	// If volumereplica is already present.
```

Comment:
removing this code can be risky, unless we know what we are doing.
consider the case:
- pool and pool-mgmt are running fine and pool is imported
- pool-mgmt gets restarted
- there won't be any import again.. right?
I guess - in those cases, below code will be useful.